packages/app/obojobo-express/routes/api/visits.js

```diff
@@ -116,12 +117,24 @@ router
 				})
 			})
 			.then(() => {
+				return db.oneOrNone(
+					`
+						SELECT is_red_alert_enabled from red_alert_status
+						WHERE user_id = $[userId] AND draft_id = $[draftId]
```

Comment:
Note: this will turn on red alert for:
* this module used in other courses
* all versions of this module (if the instructor edits the draft, the alert persists)

Think about what it would take to isolate one or both of those conditions.

Hints: check out `draft_content_id` and `resource_link_id`